packages/js/src/workouts/tailwind-components/indexation-step.js

```diff
@@ -0,0 +1,101 @@
+import { useCallback, useState, Fragment } from "@wordpress/element";
+import { addLinkToString } from "../../helpers/stringHelpers.js";
+import { __, sprintf } from "@wordpress/i18n";
+import { ReactComponent as WorkoutStartImage } from "../../../images/motivated_bubble_woman_1_optim.svg";
+import PropTypes from "prop-types";
+import Alert from "./alert";
+import { ConfigurationIndexation } from "./ConfigurationIndexation";
+
+import AnimateHeight from "react-animate-height";
+
+/**
+ * The indexation step.
+ *
+ * @param {string}   indexingState          The indexing state.
+ * @param {Function} setIndexingState       A callback to set the indexing state.
+ * @param {boolean}  showRunIndexationAlert Whether the alert to run indexation needs to be shown.
+ *
+ * @returns {WPElement} The indexation step.
+ */
+export default function IndexationStep( { indexingState, setIndexingState, showRunIndexationAlert } ) {
```

Comment:
Why the naming of the file? `indexation-step.js` instead of `IndexationStep.js`?

Reply:
Because this file is a new file resulting from the "extraction" of the IndexationStep component out of the ConfigurationWorkout, I thought it would be better to start following the canonical _kebab-case_ file naming scheme. 🙂